src/Fixer/Basic/BracesFixer.php

```diff
@@ -809,8 +812,6 @@ private function findStatementEnd(Tokens $tokens, $parenthesisEndIndex)
                 return $tokens->getPrevNonWhitespace($index);
             }
         }
-
-        throw new \RuntimeException('Statement end not found.');
```

Comment:
?

Reply:
Unreachable statement.

Reply:
This is reachable if there's a bug in the code.

Reply:
If this line becomes reachable, PHPStan will report a missing `return` statement